gradle-failure-reports/src/main/java/com/palantir/gradle/failurereports/FailureReportsRootPlugin.java

```diff
@@ -84,12 +81,14 @@ private void collectOtherTaskFailures(Project project, TaskProvider<FinalizerTas
                                     projectTasks)
                             .filter(FailureReportsRootPlugin::isAllowedTask)
                             .filter(FailureReporterResources::executedAndFailed)
-                            .map(ThrowableFailureReporter::maybeGetFailureReport)
-                            .flatMap(Optional::stream))));
+                            .map(ThrowableFailureReporter::getFailureReport))));
         });
     }
 
     private static boolean isAllowedTask(Task task) {
-        return !(task instanceof JavaCompile) && !(task instanceof Checkstyle) && !(task instanceof FinalizerTask);
+        return !(task instanceof JavaCompile)
```

Comment:
maybe it would make sense to add support for ignoring specific tasks ?

Reply:
Maybe... we can see if it comes up as a request. I think here we're ignoring tasks we have special handling for in this plugin, so it's fine to just have a list here?